bindata/operator/operator.yaml

```diff
@@ -76,10 +76,7 @@ spec:
         env:
         - name: OPENSTACK_RELEASE_VERSION
           value: '{{ .OpenstackReleaseVersion }}'
-{{ range $envName, $envValue := .OpenStackServiceRelatedImages }}
-        - name: {{ $envName }}
-          value: {{ $envValue }}
-{{ end }}
+        envCustomImage: replace_me
```

Comment:
is this change ok?

Reply:
I dont know, if I dont run "pre-commit run -a", the pre-commit CI fails. If I run it, it adds these changes to the code.

Reply:
It still fails the pre-commit check, so removing this change